### ISSUE TITLE
fragment sum not working?

### ISSUE THREAD

**Issue by joaomcteixeira** [MDAnalysis/mdanalysis#2405]
**Behavior**

According to [documentation of unwrap()](https://www.mdanalysis.org/docs/documentation_pages/core/groups.html?highlight=unwrap#MDAnalysis.core.groups.AtomGroup.unwrap):

```
An AtomGroup containing all atoms of all fragments in the group ag can be created with:

all_frag_atoms = sum(ag.fragments)
```

I have an Atom selection with 5 different chains, with the following output:

```
print(selection.fragments)
(<AtomGroup with 4052 atoms>, <AtomGroup with 4052 atoms>, <AtomGroup with 4052 atoms>, <AtomGroup with 965 atoms>, <AtomGroup with 938 atoms>)
```

after doing:

```
all_frag_atoms = sum(selection.fragments)                                   
print(all_frag_atoms.fragments)
```

I got the same output:

```
(<AtomGroup with 4052 atoms>, <AtomGroup with 4052 atoms>, <AtomGroup with 4052 atoms>, <AtomGroup with 965 atoms>, <AtomGroup with 938 atoms>)
```

**Question?**

Is this the expected behavior? If yes, could you explain briefly the rationale I am missing?

**Version**

I am running the latest develop build: https://github.com/MDAnalysis/mdanalysis/commit/615b86b6fb3bf591289ededb8408ecbcf7059309
python: 3.7.3

**Comment by lilyminium** [comment]
[Fragments are AtomGroups where each atom is bonded to at least one other atom in the group](https://www.mdanalysis.org/UserGuide/groups_of_atoms.html#fragments). They are automatically calculated by MDAnalysis. The only way to change the fragments in a Universe is by adding bonds. 

`all_frag_atoms = sum(selection.fragments)` gives you an AtomGroup of all the atoms in `selection.fragments` (which might be equal to, or contain more atoms, than `selection`). This means that `all_frag_atoms == sum(all_frag_atoms.fragments)` is `True`. 

Creating a new AtomGroup doesn't change the bonding and therefore does not change the fragments. The following is always true:
```
ag.fragments == sum(ag.fragments).fragments
```
What behaviour were you expecting / trying to get?

**Comment by zemanj** [comment]
@lilyminium I don't see any problem in the code since what the OP gets is exactly what's expected. However, the phrasing
> Fragments are AtomGroups where each atom is bonded to at least one other atom in the group.

is not quite right, since it would also apply to a group of fragments. It should read

"A fragment is an AtomGroup where any atom is reachable from any other atom in the AtomGroup by traversing bonds, and none of its atoms is bonded to any atoms outside the AtomGroup."

**Comment by lilyminium** [comment]
Thanks for the clarification!

**Comment by orbeckst** [comment]
@joaomcteixeira does this make things clearer? If so, we can close this issue (or do you want to leave it open for clarifying docs, @lilyminium ?)

**Comment by joaomcteixeira** [comment]
I understand, I missed that strong restriction on *bonds between atoms*.

The context of the question was the following:
when running unwrap in the sum of fragments, I did got whole molecules but the relative position between molecules (chains) was simply lost. Initially I was expecting the sum of fragments to solve the issue because I thought this would unite all fragments in a single unit, so to speak. I then solved the problem by applying `reference='com'`.

@orbeckst @lilyminium @zemanj thanks for the clarification, you can close the issue if you wish, I leave it open in case you want to maintain the need for updating the docs.

**Comment by lilyminium** [comment]
I've clarified the docs with @zemanj's more detailed explanation, thanks! Feel free to close it :)

**Comment by orbeckst** [comment]
Then can you please put in your commit a “fixes #2405” so that merging autocloses this issue? Thanks.
